proto/osmosis/superfluid/tx.proto

```diff
@@ -231,4 +236,42 @@ message MsgAddToConcentratedLiquiditySuperfluidPositionResponse {
     (gogoproto.nullable) = false
   ];
   uint64 lock_id = 4 [ (gogoproto.moretags) = "yaml:\"lock_id\"" ];
+}
+
+// ===================== MsgUnbondConvertAndStake
+message MsgUnbondConvertAndStake {
+  option (amino.name) = "osmosis/unbond-convert-and-stake";
+
+  // lock ID to convert and stake.
+  // lock id with 0 should be provided if converting liquid gamm shares to stake
+  uint64 lock_id = 1 [ (gogoproto.moretags) = "yaml:\"lock_id\"" ];
+  string sender = 2 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
+  // validator address to delegate to.
+  // If provided empty string, we use validator returned from valset-preference
+  // module.
+  string val_addr = 3;
+  // min_amt_to_stake indicates the minimum amount to stake after conversion
+  string min_amt_to_stake = 4 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+    (gogoproto.moretags) = "yaml:\"min_amt_to_stake\"",
+    (gogoproto.nullable) = false
+  ];
+  // shares_to_convert_and_stake indicates amount of gamm shares to convert and
+  // stake.
+  cosmos.base.v1beta1.Coin shares_to_convert_and_stake = 5 [
+    (gogoproto.moretags) = "yaml:\"shares_to_convert_and_stake\"",
+    (gogoproto.nullable) = false
+  ];
```

Comment:
Oh wow, were supporting partial's here

Reply:
As talked about on slack, lets go with full conversion only for now

Reply:
yeah, this field was originally added with the intention of 

- being able to add partial logic easier
- supporting partial stake of liquid lp tokens. 

But on the second thought, deleting this field for now